### DISCUSSION TITLE
Home Assistant - Energy Dashboard - Water Meter

### DISCUSSION THREAD

**Discussion by seaverd** [allangood/rtlamr2mqtt#177] [Q&A] [answered]
So I have been successfully using this docker to read my Neptune R900 water meter for several months.  I see that the November release of Homeassistant now supports water sources in the energy dashboard.  When I click add water source and then attempt to add a sensor, no sensors are available...see attached screenshot.

![watersource](https://user-images.githubusercontent.com/2743685/201547573-892c82f7-2cb7-4aea-94e3-ebcf5d86cb81.JPG)

The link for the help lead to this explanation, see attached screenshot:

![watersource_2](https://user-images.githubusercontent.com/2743685/201547743-006e2466-bb1c-40c7-b547-e4ee16baf33b.JPG)

Any suggestions, I am running in docker, so is there something I need to configure differently?

**Comment by allangood** [comment, marked as answer]
You have to change the `device_class` setting from the default `none` to `water`

Regards

**Comment by seaverd** [comment]
Thank you!  Worked perfectly.

Dan

**Reply by allangood** [reply]
Glad it is working!

**Comment by seaverd** [comment]
Now I’m off to research where the data is stored and if it can be edited.
Somehow in August I used -85,000 gallons.  So there must be some bad
datapoint in there.  All other months make sense to me.

Dan

**Reply by par-texx** [reply]
https://www.reddit.com/r/homeassistant/comments/yupypw/rtlamr_and_my_adventure_getting_it_running/

Just wrote about that problem (bad data), and how I fixed it on reddit.
